tests/conftest.py

```diff
@@ -87,13 +98,20 @@ def backend(request):
     only_backends = [
         pid for pid in param_ids if request.node.get_closest_marker(f'only_{pid}')
     ]
+    disable_backend = any(
+        backend in param_id for backend in request.config.option.disable_backend
+    )
 
     if skip_backend and (param_id in only_backends):
         raise ValueError(
             f"Must specify skip_{param_id} or only_{param_id} but not both!"
         )
 
-    if skip_backend:
+    if disable_backend:
+        pytest.skip(
+            f"skipping {func_name} as the backend is disabled via --disable-backend"
```

Comment:
```suggestion
            f"skipping {func_name} as the backend is disabled via --disable-backend {param_id}"
```

While technically redudnant information, this would give the exact command information that was used:

```console
$ pytest tests/test_tensor.py --disable-backend tensorflow
...
SKIPPED [1] tests/test_tensor.py:550: skipping test_tensor_precision[tensorflow] as the backend is disabled via --disable-backend tensorflow
...
```

Reply:
Not correct, because param_id becomes `numpy_minuit` in some cases, which is only disabled if `--disable-backend numpy` or `--disable-backend minuit`

Reply:
Ah very good point, sorry I missed that. The following would work though

```suggestion
            f"skipping {func_name} as the backend is disabled via "
            + " ".join(
                [
                    f"--disable-backend {choice}"
                    for choice in request.config.option.disable_backend
                ]
            )
```

as that would allow for

```console
$ pytest tests/test_tensor.py --disable-backend tensorflow
...
SKIPPED [1] tests/test_tensor.py:550: skipping test_tensor_precision[tensorflow] as the backend is disabled via --disable-backend tensorflow
...
```

as well as

```console
$ pytest tests/test_tensor.py --disable-backend tensorflow --disable-backend minuit
...
SKIPPED [1] tests/test_tensor.py:550: skipping test_tensor_precision[tensorflow] as the backend is disabled via --disable-backend tensorflow --disable-backend minuit
SKIPPED [1] tests/test_tensor.py:550: skipping test_tensor_precision[numpy_minuit] as the backend is disabled via --disable-backend tensorflow --disable-backend minuit
...
```

What do you think?